server/src/main/java/org/elasticsearch/search/aggregations/MultiBucketConsumerService.java

```diff
@@ -52,27 +53,35 @@ private void setMaxBucket(int maxBucket) {
 
     public static class TooManyBucketsException extends AggregationExecutionException {
         private final int maxBuckets;
+        private final int bucketsCount;
```

Comment:
Maybe this rename to `bucketCount`?

Reply:
Since the other field is called `maxBuckets` with a final `s` I used the `s` here for consistency...anyway.